src/main/java/clients/TestTypesClient.java

```diff
@@ -89,6 +89,7 @@ private Response callTestTypes(String id, TestTypeQueryParam testTypeQueryParam)
     private Response callGetTestTypes(Filter filter) {
         Response response = given().filters(filter)
                 .contentType(ContentType.JSON)
+                .log().all()
```

Comment:
This won't put bearer tokens into the debug will it?  That shouldn't be in there.

Reply:
fixed